modules/openxr/action_map/openxr_interaction_profile.h

```diff
@@ -41,26 +41,30 @@ class OpenXRIPBinding : public Resource {
 
 private:
 	Ref<OpenXRAction> action;
-	PackedStringArray paths;
+	String source_path;
+
+	// PackedStringArray paths;
 
 protected:
 	static void _bind_methods();
 
 public:
-	static Ref<OpenXRIPBinding> new_binding(const Ref<OpenXRAction> p_action, const char *p_paths); // Helper function for adding a new binding
+	static Ref<OpenXRIPBinding> new_binding(const Ref<OpenXRAction> p_action, const String &p_source_path); // Helper function for adding a new binding
 
 	void set_action(const Ref<OpenXRAction> p_action); // Set the action for this binding
 	Ref<OpenXRAction> get_action() const; // Get the action for this binding
 
-	int get_path_count() const; // Get the number of io paths
+	void set_source_path(const String &path);
+	String get_source_path() const;
+
+	// Deprecated
```

Comment:
We can't remove them all because `set_paths` and `get_paths` are still needed to convert old action maps to new. But yeah I can add that.

Reply:
I think if someone made a custom `deprecated=no` build, then it shouldn't be able to convert old action maps, only load those with the current format.

With a GDExtension that uses a deprecated method, loading it will actually fail in a `deprecated=no` build. I feel like failing to load old action maps is more or less equivalent?